ci-scripts/JenkinsFile-CWAG-integ-test

```diff
@@ -0,0 +1,173 @@
+#!/bin/groovy
+
+def GIT_URL = "github.com/magma/magma"
+def GIT_BRANCH = "master"
+def slack_channel = "#magma-ci-bot"
+
+pipeline {
+  agent {
+      label "CWAG"
+  }
+  options {
+    timestamps()
+    ansiColor('xterm')
+  }
+  stages {
+    stage("Prepate Env"){
+      steps{
+        script {
+          echo "Clean Env"
+          cleanWs()
+          echo "Clone Sources"
+          checkout(
+            changelog: false, 
+            poll: false, 
+            scm: [$class: 'GitSCM', 
+                  branches: [[name: '${sha1}']], 
+                  doGenerateSubmoduleConfigurations: false, 
+                  extensions: [], 
+                  submoduleCfg: [], 
+                  userRemoteConfigs: [[refspec: '+refs/pull/*:refs/remotes/origin/pr/*', url: "https://" + GIT_URL + ".git"]]]
+          )
+        }
+      }
+    }
+    stage("cwag-precommit"){
+      environment {
+        MAGMA_ROOT = sh(script: 'pwd', , returnStdout: true).trim()
+        GO111MODULE = "on"
+        GOPROXY = "https://proxy.golang.org"
+      }
+      steps{
+        script {
+          sh "./circleci/golang_before_install.sh"
+          dir("cwf/gateway"){
+            sh "go mod download"
+            sh "make -C ${MAGMA_ROOT}/cwf/gateway precommit"
+            sh "make -C ${MAGMA_ROOT}/cwf/gateway/integ_tests precommit"
+        }
+        }
+      }
+    }
+    stage("cwf-integ-test"){
+      environment {
+        CIRCLE_REPOSITORY_URL = "https://${GIT_URL}.git"
+        CIRCLE_BRANCH = "${GIT_BRANCH}"
+        CIRCLE_SHA1 = "HEAD"
+        PYTHONPATH = "${WORKSPACE}/orc8r"
+      }
+      steps{
+        script {
+          echo "Setup required packages"
+          sh """
+            export PATH;
+            export PYENV_ROOT="\$HOME/.pyenv"
+            export PATH="\$PYENV_ROOT/bin:\$PATH"
+            eval "\$(pyenv init -)"
+            eval "\$(pyenv virtualenv-init -)"
+            pyenv global 3.7.0
+            pip3 install --upgrade pip
+            pip3 install fabric3 jsonpickle requests PyYAML awscli docker-compose
+          """
+          sh "vagrant global-status 2>/dev/null | awk '/virtualbox/{print \$1}' | xargs -I {} vagrant destroy -f {}"
+          try {
+            sh "vboxmanage unregistervm cwag-dev --delete"
+          } catch (Exception e) {
+            echo "cwag-dev didn't exists as expected"
+          }
+          try {
+            sh "vboxmanage unregistervm magma-trfserver --delete"
+          } catch (Exception e) {
+            echo "magma-trfserver didn't exists as expected"
+          }
+          try {
+            sh "vboxmanage unregistervm cwag-test --delete"
+          } catch (Exception e) {
+            echo "cwag-test didn't exists as expected"
+          }
+          dir('cwf/gateway') {
+            dir('docker') {
```

Comment:
Question: What's the point to do 2 following dir()  instead of one ? `dir('cwf/gateway/docker')`

Reply:
there will be 2 dir statements in any case cause when `dir("docker"){...}` block ends we have few commands executed in  `dir('cwf/gateway')` block.
Other way to do same operations is to do:
```
dir('cwf/gateway'/docker){....}
dir('cwf/gateway'){....}
```